CHANGES.md

```diff
@@ -9,103 +15,523 @@ First release candidate for Chapel 2.0 (RC1)
 ---------------------------------------------
+* top-level modules must now be `use`d in order to be referenced
+  (see TODO blc)
+* class types must now opt into being able to store `nil` using `?`
+  (e.g., `var myC: C = ...` cannot store `nil` but `var myC: C? = ...` can;
+   see https://chapel-lang.org/docs/1.20/language/evolution.html#nilability-changes)
+* for a class `C`, `new C(...)` is now equivalent to `new owned C(...)`
+  (see https://chapel-lang.org/docs/1.20/language/evolution.html#new-c-is-owned)
+* for a class `C`, un-decorated `C` now means non-nil and generic management
+  (see https://chapel-lang.org/docs/1.20/language/evolution.html#undecorated-classes-have-generic-management)
+* passing owned/shared objects to untyped formals no longer changes their types
+  (see https://chapel-lang.org/docs/1.20/language/evolution.html#new-default-intent-for-owned-and-shared)
+* arguments declared with owned/shared types now default to `const ref` intent
+  (see https://chapel-lang.org/docs/1.20/language/evolution.html#new-default-intent-for-owned-and-shared)
+* string indexing, iteration, and lengths now use codepoint units by default
+  (see https://chapel-lang.org/docs/1.20/builtins/String.html#String.string.length)
+* `string.find` now returns `byteIndex` values
+* `void` is no longer allowed as the type of a variable (use `nothing` instead)
+  (see 'The Void Type' section in the 'Types' chapter of the spec)
+* made `#0` preserve the given indices of a range or domain
+  (e.g., `5..#0` now returns `5..4` rather than `1..0`)
+* disabled support for assigning from ranges to multidimensional arrays
+  (see 'Array Assignment' in the `Arrays` chapter of the language spec)
+* made slice expressions be governed by their slicing domain
+  (e.g., in `ref B = A[D];` the domain of `B` is `D` rather than a copy of it)
+* made `proc foo(x: bool(?))` generic across all bool sizes including `bool`
+  (see TODO blc)
+* `require` statements are now processed relative to the source file directory
+  (see TODO blc)
+* class downcasts now throw `ClassCastError` for incompatible runtime types
+  (see 'Explicit Class Conversions' in the spec
+   and https://chapel-lang.org/docs/1.20/language/evolution.html#readme-evolution-nilability-and-casts)
+* casts from a C pointer type to a class type now require a nilable class
+  (see https://chapel-lang.org/docs/1.20/language/evolution.html#readme-evolution-nilability-and-casts)
+* `isSubtype()` and comparison operators on types no longer consider coercions
+  (see https://chapel-lang.org/docs/1.20/builtins/UtilMisc_forDocs.html#UtilMisc_forDocs.isSubtype)
 
 New Features
 ------------
+* added support for `private` and `public` `use` statements
+  (see 'The Use Statement' in the 'Statements' chapter of the language spec)
+* added a built-in `bytes` type
+  (see https://chapel-lang.org/docs/1.20/builtins/Bytes.html)
+* added bytes literals `b""`, `b''`, `b""" """`, and `b''' '''`
+  (see TODO)
```

Comment:
If you meant this TODO as "add link to doc", then the Bytes doc page have a section about creating bytes which alludes to these literals. The link is the same as the one above:
> (see https://chapel-lang.org/docs/1.20/builtins/Bytes.html)

If you wanted to add an example, something like this should do IMO:
> (e.g. in `var myVar = b"my bytes";`, `myVar` is a `bytes`)